frappe/custom/doctype/customize_form/customize_form.json

```diff
@@ -18,6 +19,7 @@
   "track_changes",
   "track_views",
   "allow_auto_repeat",
+  "allow_merge_with_existing",
```

Comment:
is the `with_existing` really needed? Wouldn't `allow_merge` describe the context well enough?